skyportal/handlers/api/internal/recent_sources.py

```diff
@@ -4,6 +4,7 @@
 from baselayer.app.access import auth_or_token
 from ...base import BaseHandler
 from ....models import Obj, Source
+from .source_views import tIndex
```

Comment:
Camelcase in Python land!  Should become t_index

Reply:
Yes, agreed (I didn't author this, just moved it around, as you can see from the diff -- will update anyways).

Reply:
👍